SkycoinProject/skywire-mainnet
# [WIP] add cli flag to use testingdeployment

Fixes #37 	

 Changes:	
-	adds flag to use Test deploymeny skywire.cc

How to test this PR:

- make build; make install

- skywire-cli node gen-config -t ==> should create config with skywire.cc deployment

- skywire-cli node gen-config -r ==> should create config with skycoin.com deployment
